### ISSUE TITLE
otlpmetrichttp not tagged with v0.21.0

### ISSUE THREAD

**Issue by seanschade** [open-telemetry/opentelemetry-go#2076]
### Description

It looks like the `otlpmetrichttp` module wasn't tagged with `v0.21.0`.
Go will pull in the latest, but I need to pin to a specific version.

### Environment

- OS: [macOS]
- Architecture: [amd64, darwin]
- Go Version: [1.16]
- opentelemetry-go version: [v0.21.0]

### Steps To Reproduce

1. `go get go.opentelemetry.io/otel/exporters/otlp/otlpmetric/otlpmetrichttp@v0.21.0`
2. See error:

```
go.opentelemetry.io/otel/exporters/otlp/otlpmetric/otlpmetrichttp@v0.21.0: reading go.opentelemetry.io/otel/exporters/otlp/otlpmetric/otlpmetrichttp/exporters/otlp/otlpmetric/otlpmetrichttp/go.mod at revision exporters/otlp/otlpmetric/otlpmetrichttp/v0.21.0: unknown revision exporters/otlp/otlpmetric/otlpmetrichttp/v0.21.0
```

### Expected behavior

I would expect all modules to be tagged with a version.

**Comment by Aneurysm9** [comment]
This module was added after the `v0.21.0` release and will be included in the `v0.22.0` release I expect will be shipped next week.

**Comment by seanschade** [comment]
Thanks @Aneurysm9 right now I am having a difficult time using this package in conjunction with `otlpmetricgrpc`. 

```
# go.opentelemetry.io/otel/exporters/otlp/otlpmetric/otlpmetrichttp
../../../../go/pkg/mod/go.opentelemetry.io/otel/exporters/otlp/otlpmetric/otlpmetrichttp@v0.0.0-20210708203527-484258eb36a7/client.go:91:8: cfg.MaxAttempts undefined (type otlpconfig.Config has no field or method MaxAttempts)
../../../../go/pkg/mod/go.opentelemetry.io/otel/exporters/otlp/otlpmetric/otlpmetrichttp@v0.0.0-20210708203527-484258eb36a7/client.go:92:6: cfg.MaxAttempts undefined (type otlpconfig.Config has no field or method MaxAttempts)
../../../../go/pkg/mod/go.opentelemetry.io/otel/exporters/otlp/otlpmetric/otlpmetrichttp@v0.0.0-20210708203527-484258eb36a7/client.go:94:8: cfg.MaxAttempts undefined (type otlpconfig.Config has no field or method MaxAttempts)
../../../../go/pkg/mod/go.opentelemetry.io/otel/exporters/otlp/otlpmetric/otlpmetrichttp@v0.0.0-20210708203527-484258eb36a7/client.go:95:6: cfg.MaxAttempts undefined (type otlpconfig.Config has no field or method MaxAttempts)
../../../../go/pkg/mod/go.opentelemetry.io/otel/exporters/otlp/otlpmetric/otlpmetrichttp@v0.0.0-20210708203527-484258eb36a7/client.go:97:8: cfg.Backoff undefined (type otlpconfig.Config has no field or method Backoff)
../../../../go/pkg/mod/go.opentelemetry.io/otel/exporters/otlp/otlpmetric/otlpmetrichttp@v0.0.0-20210708203527-484258eb36a7/client.go:98:6: cfg.Backoff undefined (type otlpconfig.Config has no field or method Backoff)
../../../../go/pkg/mod/go.opentelemetry.io/otel/exporters/otlp/otlpmetric/otlpmetrichttp@v0.0.0-20210708203527-484258eb36a7/client.go:160:30: d.generalCfg.MaxAttempts undefined (type otlpconfig.Config has no field or method MaxAttempts)
../../../../go/pkg/mod/go.opentelemetry.io/otel/exporters/otlp/otlpmetric/otlpmetrichttp@v0.0.0-20210708203527-484258eb36a7/client.go:177:50: d.generalCfg.Backoff undefined (type otlpconfig.Config has no field or method Backoff)
../../../../go/pkg/mod/go.opentelemetry.io/otel/exporters/otlp/otlpmetric/otlpmetrichttp@v0.0.0-20210708203527-484258eb36a7/client.go:186:85: d.generalCfg.MaxAttempts undefined (type otlpconfig.Config has no field or method MaxAttempts)
../../../../go/pkg/mod/go.opentelemetry.io/otel/exporters/otlp/otlpmetric/otlpmetrichttp@v0.0.0-20210708203527-484258eb36a7/options.go:88:23: undefined: otlpconfig.WithMaxAttempts
../../../../go/pkg/mod/go.opentelemetry.io/otel/exporters/otlp/otlpmetric/otlpmetrichttp@v0.0.0-20210708203527-484258eb36a7/options.go:88:23: too many errors
```

**Comment by emilgelman** [comment]
@seanschade experiencing the same. 
`oltpmetrichttp` is currently unusable as `otlpconfig.Config` is missing the HTTP configurations in v0.0.21:
https://github.com/open-telemetry/opentelemetry-go/blob/exporters/otlp/otlpmetric/v0.21.0/exporters/otlp/otlpmetric/internal/otlpconfig/options.go

**Comment by MrAlias** [comment]
Resolved with #2101
